html/php-challenge/good.php

```diff
@@ -0,0 +1,22 @@
+<?php
+session_start();
+require('dbconnect.php');
+
+
+
+if (!empty($_POST)) { //$_POSTが空のとき
+  //いいねする
+  // echo $_POST['post_id'];
+
+    $good = $db->prepare('INSERT INTO good SET post_id=?, member_id=?, created=NOW()');
+    $good->execute(array(
+      $_POST['id'],
+      $_SESSION['id']
+
+    ));
```

Comment:
ここは少なくとも $_POST['id'] ではなく $_POST['post_id'] が適切ですね！

```suggestion
    $good->execute(array(
      $_POST['post_id'],
      $_SESSION['id']
    ));
```

うまくいかないときは

```php
var_dump($_POST);
```

で配列の中身を調べるのが有効です